src/uwtools/config/formats/sh.py

```diff
@@ -12,17 +12,15 @@
 
 class SHConfig(Config):
     """
-    Concrete class to handle bash config files.
+    Work with key=value shell configs.
```

Comment:
We probably should mention `bash` specifically, as this is also useful for `sh`, `ksh`, `zsh`, and other Bourne-family shells.

Reply:
Hmm, we did call it `SHConfig`, maybe we should be more explicit about the valid shells while we're at it?

Reply:
All that matters is that the shell understands `key=value` variable assignment. There are a LOT of shells that this applies to -- too many to usefully try to enumerate here, I think. It's even potentially usable with a number of non-shell languages, e.g.
```
$ cat vars.py
a="apple"
pi=3.14
$ python -c "import vars; print(vars.a, vars.pi)"
apple 3.14
```

Reply:
Oh, certainly. I mean "...bash and other..." or "...any Bourne shell..." to emphasize that the 'sh' naming is more inclusive, but its not really necessary and probably over-complicating